deploy/templates/components/manager/manager.yaml

```diff
@@ -46,4 +46,4 @@ spec:
           - name: RELATED_IMAGE_default_tls_secrets_creation_job
             value: "quay.io/eclipse/che-tls-secret-creator:alpine-3029769"
           - name: RELATED_IMAGE_pvc_cleanup_job
-            value: "quay.io/libpod/busybox:1.30.1"
+            value: "registry.access.redhat.com/ubi8-minimal:8.3-291"
```

Comment:
Should we seperate this out into a seperate commit so that if we ever have to go looking for when it was changed it might be easier?

Reply:
Good catch. I did not even plan to change it in the current PR, but seems it was made and I accidentally committed it.

After we do milestone 1, I'll update PR and make sure generated stuff are separate from other and test if it works. And then only re-request review.